docs/DOCKER.md

```diff
@@ -112,6 +112,47 @@ Using VSCODE
 -----------------------
 In order to use VSCODE in debugging mode with DOCKER, check out [VSCODE_CONFIGURATION](VSCODE_CONFIGURATION.md)
 
+## Working on anyway-etl and Airflow
+
+Anyway ETL processes and Airflow server are developed in a different repository: [hasadna/anyway-etl](https://github.com/hasadna/anyway-etl)
+but we use the same Docker Compose environment.
+
+For some tasks you will also need to set secret values in the `.env` file, ask a team member for these values.
```

Comment:
I suggest we'll create a unique env file for the ones who need it, and add a link to it (not an open link rather a link that requires access request - similar to what we have today)

Reply:
I agree

Reply:
opened an issue for documentation improvements: #1877